### PR TITLE
West Midlands Class 4 - Saadia El fekak - JS Core 1 - Week 1

### PR DESCRIPTION
**Pull request by SaadiaELF**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Saadia El fekak
- Your City: Birmingham
- Your Slack Name: SaadiaELF

# Homework Details

- Module: JS-Core-1
- Week: 1

# Notes

- What did you find easy?
Mandatory and exercises folders
- What did you find hard?
3-magic-8-ball extra activity , the instructions wasn't clear. And one test is not working.
- What do you still not understand?

- Any other notes?

### CODE REVIEW

**Review by Vanipriyaarun — COMMENTED** [review]
Hi, I am wondering why lines 151-156 are commented. I am facing an error if I don't comment on these lines, otherwise, mine works fine. I could see these lines are to check if the checkAnswer function returns one of the outputs. I am not sure why it is not working. Thanks.

**Comment by SaadiaELF** [comment]
Hi, I commented these lines because there is an error in the test. Although the code is correct the test is failing.
Refer to James message on Slack about it : https://codeyourfuture.slack.com/archives/C03K0P5KYEB/p1659355763071149?thread_ts=1659349553.133979&channel=C03K0P5KYEB&message_ts=1659355763.071149

**Review by Vanipriyaarun — COMMENTED** [review]
Thank you for the clarification.

File: extra/3-magic-8-ball.js [hunk 1]
```diff
@@ -60,8 +85,47 @@ function shakeBall() {
...
+  if (positiveAns.includes(answer)) {
+    message = "positive";
+  } else if (veryPositiveAns.includes(answer)) {
+    message = "very positive";
+  } else if (negativeAns.includes(answer)) {
+    message = "negative";
+  } else {
+    message = "very negative";
+  }
+  return message;
```
Comment (DomVinyard) [inline extra/3-magic-8-ball.js:127]:
```suggestion
  if (positiveAns.includes(answer)) return 'positive'
  if (negativeAns.includes(answer)) return 'very positive'
  if (positiveAns.includes(answer)) return 'negative'
  else return 'very negative'
```

File: mandatory/1-syntax-errors.js [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
+function addNumbers(a, b, c) {
+  // the commas were missing
   return a + b + c;
 }
 
-function introduceMe(name, age)
-  return "Hello, my name is " + name "and I am " age + "years old";
-
+function introduceMe(name, age) {//the curly brackets were missing
+  return "Hello, my name is " + name + " and I am " + age + " years old"; //the plus operators were missing
```
Comment (DomVinyard) [inline mandatory/1-syntax-errors.js:9]:
```suggestion
  return `Hello, my name is ${name} and I am ${age} years old
```

File: mandatory/3-function-output.js [hunk 3]
```diff
@@ -1,16 +1,19 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getRandomNumber() {
+  // the function will return a number between 0 and 9
```
Comment (DomVinyard) [inline mandatory/3-function-output.js:3]:
```suggestion
  // the function will return a _random_ number (float) between 0 and < 10
```

**Review by DomVinyard — APPROVED** [review]
very impressive

**Pull request closed without merging**